table/model.go

```
@@ -72,3 +82,69 @@ func New(columns []Column) Model {
 func (m Model) Init() tea.Cmd {
 	return nil
 }
+
+func (m Model) isRowMatched(row Row) bool {
+	for _, column := range m.columns {
+		if column.Filterable {
```

Minor, but let's stick with fail fast pattern:

```golang
if column.Filter {
  continue
}
// ...
```

---

Nice catch.